server/views/partials/footer.njk

```diff
@@ -0,0 +1,29 @@
+{% from "govuk/components/footer/macro.njk" import govukFooter %}
+{% from "../components/feedback-banner.njk" import feedbackBanner %}
+
+  {% if hasFeedbackBanner !== false %}
+    {{
+      feedbackBanner({
+        feedbackBannerAdditionalClasses: feedbackBannerBackground
+      })
+    }}
+{% endif %}
+
+{% include "../components/feedback-banner.njk" %}
+
+{{ 
+  govukFooter({
+    meta: {
+      items: [
+        {
+          href: '/feedback-and-support',
+          text: "Get help",
+          attributes: {
```

Comment:
This "Get help" link was added to BVL but shouldn't be in WPIP... so this govukFooter should be removed I think